apps/web-giddh/src/assets/locale/all-items/en.json

```diff
@@ -552,6 +552,34 @@
                             ]
                         }
                     }
+                },
+                {
+                    "label": "VAT Liabilities",
+                    "link": "/pages/vat-report/liabilities",
+                    "icon": "icon-vat1",
+                    "description": "VAT Liabilities",
+                    "hide": "sidebar",
+                    "additional": {
+                        "queryParams": {
```

Comment:
As confirmed with kriti please do for voucher version 2 only

Reply:
only for 2